testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -338,8 +345,22 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
     }
 
     void usePluginVersion(String version) {
+        def develocityPlugin = VersionNumber.parse(version) >= VersionNumber.parse("3.17")
         def gradleEnterprisePlugin = VersionNumber.parse(version) >= VersionNumber.parse("3.0")
-        if (gradleEnterprisePlugin) {
+        if (develocityPlugin) {
```

Comment:
🤔 Do we need smoke tests for the Gradle Enterprise plugin 3.17 as well? Or is it enough to test the Develocity plugin 3.17?

Reply:
I don't see much value in doing this.